scripts/check-system.sh

```diff
@@ -14,7 +14,7 @@ done
 
 if which docker &> /dev/null; then
     docker_host_memory=$(docker info --format "{{.MemTotal}}")
-    if [ ${docker_host_memory} -gt $((8 * 10 ** 9)) ]; then
+    if [ ! ${docker_host_memory} -gt $((8 * 10 ** 9)) ]; then
```

Comment:
Quick question: Shouldn't this be 

```sh
if [ ${docker_host_memory} -lt $((8 * 10 ** 9)) ]; then
```

(otherwise, since ¬(a > b) ≡ (a ≤ b) users would get the warning even if they had configured exactly 8 GB)?